homeassistant/components/iperf3/__init__.py

```diff
@@ -48,16 +49,22 @@
 ATTR_VERSION = "Version"
 ATTR_HOST = "host"
 
+ICON = "mdi:speedometer"
+
 SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
     SensorEntityDescription(
         key=ATTR_DOWNLOAD,
         name=ATTR_DOWNLOAD.capitalize(),
+        icon=ICON,
```

Comment:
Why was this moved? @yuvalabou?

Reply:
My apologize, pushed too early - see #85809